src/common/autoSave/AutoSave.tsx

```diff
@@ -2,9 +2,9 @@ import React, { Component } from 'react';
 import { FormSpy, FormSpyRenderProps } from 'react-final-form';
 
 interface Props {
-  save: Function;
+  save: (values: any) => void;
   debounce: number;
-  values?: FormData;
+  values?: any;
```

Comment:
Is it possible to avoid `any` here?

Reply:
Not with the current form types, I'm afraid. I did make it generic with `any` and figured out how to make AutoSave work as a function component though. Added new commits for those.